lib/commands/preload.ts

```diff
@@ -169,6 +169,14 @@ Can be repeated to add multiple certificates.\
 		try {
 			const fs = await import('fs');
 			await fs.promises.access(params.image);
+			const path = await import('path');
```

Comment:
There are other ways to get the file extension without importing path (if desired). LMK